db/migrate/20190807163019_create_venues.rb

```diff
@@ -0,0 +1,12 @@
+# :reek:UncommunicativeVariableName
+class CreateVenues < ActiveRecord::Migration[5.2]
+  def change
+    create_table :venues do |t|
+      t.string :address, null: false, unique: true
```

Comment:
please always provide default value alongside with `null: false`. Maybe it's not relevant currently, but previously this caused annoying problems((

Reply:
resolved